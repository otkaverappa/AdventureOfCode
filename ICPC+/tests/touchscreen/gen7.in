19
ovmmhcu 10
rvpmzcz
mmrjtvp
poevynw
nkudiir
rloiidw
iivwwnh
pkygadd
yszoevt
melerzo
zeljogi
ltpcdoj 10
etmenos
tjmgpzm
gaaignt
vubiptk
vjvzekd
ryykjms
zlghlhp
rwjoqmw
ifjdogc
sqhlqhy
atqbgxk 10
rgalclz
vxydfjg
tpkogtz
ybxyuqb
bnmuvoh
xzjsyhx
dqdzfnn
nhmmimm
ecqhpce
msljrue
rdbvvhw 10
dujqdve
ljrslhb
clfodta
wpepthm
ooirjrk
opoayhu
kqxmdfd
gydenhu
iqgzfpq
qicexrg
waciraw 10
ufzdfcj
slfdcoc
jfubnxf
lqojqqt
jsrgzrl
evwzjcc
nqeyxaz
kxgxpui
inbrftz
fmkjjjk
ulmhdsf 10
atekslk
khssuul
cplhdxs
ogdjrrq
wmywfej
zrtjznd
vjpxaag
dxbseeb
xxtwjru
qydqpyb
qmhmxwl 10
ywsbvtv
bzybwsx
glryjxq
bxsrlbf
jxrhvjk
segudex
byujkni
vkywisq
vtveqml
lxygbga
gkxiltr 10
xizstyr
dqjymef
ctsqqqw
uwyajxi
wsatabn
vzeaspz
eugipyy
hrwbpue
yrmujop
mqdhrji
jzjpvpy 10
mpwtgvx
ypbwjpt
ugkgynq
pxybyhq
vyohnna
wizuzas
ksnhyyp
wnfnmfo
kmhhmvo
akryuqs
utmglan 10
lafjnkw
cqnoeuw
rrktcbr
wslslxz
yzojbtu
restwfh
azdtrqm
vrdrlqk
yolypzj
rtdkzwd
xbmxdst 10
wifrblk
pbwnrjm
jjvcebm
fxrcadb
fvubfcu
hnhyqdl
knztyxx
cajhxal
zfmfdhi
kjfrymr
qrdaeev 10
edsiddr
cdcclpj
qyrahyu
hklxcqy
ivvnyov
erpgxrk
kgtaena
nluuyiu
aysitpx
udvaxkg
wbrgkmg 10
qbigpfa
nnunlox
hgxdjsd
ielehck
rrsksdq
hiswxpj
kfjtlgw
vabfenk
lrwfirp
duhmezj
dounufh 10
flfcnhi
tuuhnqm
yhdbdmp
hlymcva
wahdooi
bvqxrke
fcsdkxe
pjvzwul
yrlurua
fjijeag
ykmdmeg 10
ydmqmhp
idahumd
miftrne
ynmwzyb
oehohte
vctggup
aiuosak
mpokfdi
ebjuisj
rnpppjv
vfkypeo 10
hgatwpf
bsphvac
fsnxfcm
xnjuuvu
jbjtijo
gauhulq
smsxgfw
okklztf
uqafsla
cwpixlr
rwhjkcj 10
rjfftqs
tlzpcbw
umxyjmg
gzzzvgl
gkuzuze
nrwidxz
fzwblvz
wjheiie
dqrlblk
vmqkeov
jmuqlst 10
ynuvwbz
elfkbwv
cjixyyi
endnbzf
msymhsh
evjkgou
hnskwaj
uarypwn
qwseqqr
xiyddho
lyitlcd 10
jcmefge
ueuncmr
sdisnjv
tsleswz
dycndrt
izzedtr
gglalvu
aesvzdc
rzbxagm
dxhnygr
