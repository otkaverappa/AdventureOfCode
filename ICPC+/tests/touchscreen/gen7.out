mmrjtvp 19
melerzo 23
pkygadd 23
yszoevt 23
rvpmzcz 24
zeljogi 25
poevynw 26
iivwwnh 27
rloiidw 27
nkudiir 30
etmenos 23
ifjdogc 26
vjvzekd 26
ryykjms 30
sqhlqhy 32
rwjoqmw 34
tjmgpzm 35
vubiptk 35
gaaignt 36
zlghlhp 38
dqdzfnn 21
ecqhpce 22
ybxyuqb 24
vxydfjg 27
nhmmimm 29
rgalclz 30
bnmuvoh 33
xzjsyhx 33
tpkogtz 37
msljrue 38
iqgzfpq 18
gydenhu 19
kqxmdfd 20
dujqdve 21
qicexrg 23
clfodta 24
ljrslhb 27
opoayhu 32
wpepthm 32
ooirjrk 33
nqeyxaz 18
lqojqqt 26
ufzdfcj 26
evwzjcc 27
inbrftz 27
jfubnxf 29
jsrgzrl 31
slfdcoc 32
fmkjjjk 35
kxgxpui 39
cplhdxs 14
ogdjrrq 21
vjpxaag 21
wmywfej 22
dxbseeb 24
zrtjznd 28
khssuul 31
atekslk 32
xxtwjru 35
qydqpyb 36
segudex 22
bzybwsx 26
jxrhvjk 26
byujkni 27
vtveqml 28
ywsbvtv 29
glryjxq 31
lxygbga 33
vkywisq 33
bxsrlbf 38
eugipyy 14
hrwbpue 18
xizstyr 19
dqjymef 24
yrmujop 25
mqdhrji 29
wsatabn 29
ctsqqqw 30
uwyajxi 35
vzeaspz 38
ksnhyyp 22
kmhhmvo 27
pxybyhq 28
ugkgynq 28
ypbwjpt 29
vyohnna 33
wnfnmfo 33
mpwtgvx 39
akryuqs 41
wizuzas 43
yolypzj 15
vrdrlqk 17
wslslxz 22
rrktcbr 23
restwfh 24
rtdkzwd 26
yzojbtu 26
azdtrqm 27
lafjnkw 31
cqnoeuw 36
zfmfdhi 13
fvubfcu 14
fxrcadb 17
cajhxal 20
jjvcebm 22
hnhyqdl 23
kjfrymr 29
knztyxx 29
wifrblk 29
pbwnrjm 32
qyrahyu 16
edsiddr 17
udvaxkg 21
kgtaena 22
aysitpx 23
erpgxrk 23
ivvnyov 26
hklxcqy 27
cdcclpj 28
nluuyiu 38
qbigpfa 16
hgxdjsd 21
nnunlox 21
kfjtlgw 22
vabfenk 23
duhmezj 26
lrwfirp 26
rrsksdq 27
ielehck 28
hiswxpj 31
fjijeag 15
tuuhnqm 15
flfcnhi 17
hlymcva 18
yrlurua 27
fcsdkxe 28
wahdooi 28
yhdbdmp 28
bvqxrke 31
pjvzwul 33
ynmwzyb 15
ydmqmhp 18
ebjuisj 20
miftrne 24
mpokfdi 24
oehohte 25
idahumd 27
aiuosak 28
vctggup 29
rnpppjv 30
jbjtijo 15
xnjuuvu 15
fsnxfcm 25
bsphvac 26
gauhulq 26
cwpixlr 31
hgatwpf 33
okklztf 34
smsxgfw 34
uqafsla 44
dqrlblk 20
fzwblvz 21
rjfftqs 24
umxyjmg 24
nrwidxz 26
wjheiie 26
gkuzuze 27
gzzzvgl 28
tlzpcbw 33
vmqkeov 34
msymhsh 21
cjixyyi 22
uarypwn 22
ynuvwbz 26
endnbzf 27
xiyddho 27
hnskwaj 28
elfkbwv 29
evjkgou 31
qwseqqr 34
gglalvu 19
jcmefge 21
ueuncmr 23
sdisnjv 26
dxhnygr 27
tsleswz 27
dycndrt 28
aesvzdc 32
izzedtr 32
rzbxagm 39
